test/integration/query-interface.test.js

```diff
@@ -112,6 +113,114 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
         expect(tableNames).to.include('my_test_table2');
       });
     }
+
+    if (dialect.supports.schemas) {
+      describe('schema support tests', () => {
```

Comment:
we're migrating tests out of this file (slowly). Can you add these tests to `test/integration/query-interface/show-all-tables.ts` instead?

Reply:
1. https://github.com/sequelize/sequelize/pull/15101/commits/b631bbef6fe32dced5a36b49e31f27410d872c49
2. https://github.com/sequelize/sequelize/pull/15101/commits/9fdc716a896042d26ee056c15e9923cbb4c6beea